submissions/boladeMarindoti/html/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>E-commerce Site</title>
+</head>
+<body>
+    <img src="./images/bells logo.png" id="logo" alt="bells logo">
+    <h1 style="text-align:center;">BELLS FEMALE APPAREL STORE</h1>
+    <nav>
+    <uL>
+     <li><a href="index.html">Home</a></li>
+     <li><a href="about.html">About</a></li>
+     <li><a href="contact.html">Contact</a></li>
+    </ul>
+    </nav>
+    <h2>Featured Products</h2>
+    <br>
+    <!--green flare dress-->
+    <h3>Green flare dress</h3>
+    <img src="./images/flare dress.jpg" alt="green flare dress" width="250px" height="250px">
+    <p>&#8358;5000</p>
+    <h2>Product detail</h2>
+    <p>This Skater Dress is designed in green irresistible midi length with attractive  3/4 sleeve and a black trimming details on the  neck, burst and sleeve to give it a unique detail. Visit our store to choose from our collections which will give you the poise and elegant look you deserve. Paired with right accessories it can be worn to different occasions. This dress is made from solid quality fabric -spandex. It is a little bit below the knee. It suite all body shapes and size. It comes  in different UK SIZES 10, 12, 14, 16, 18. </p>
+    <!--female unique gown-->
+    <h3>Female unique gown</h3>
+    <img src="./images/female unique gown.jpg" alt="female unique gown" width="250px" height="250px">
+    <p>&#8358;4500</p>
+    <h2>Product detail</h2>
+    <p>The female unique dress is a tube gown that can be worn as a  casual or co-operate dress depending on how it is styled.It comes in different sizes ranging from UK sizes 8,10,12,14,16,18.</p>
+    <!--Fancy Finery Alisa Sleeveless Bodycon-->
+    <h3>Fancy Finery Alisa Sleeveless Bodycon</h3>
+    <img src="./images/alicia dress.jpg" alt="Fancy Finery Alisa Sleeveless Bodycon" width="250px" height="250px">
```

Comment:
You're meant to use ONLY 2 product images from the instructions provided. You need to update your files to that.